src/activities/shapes/Shapes.qml

```diff
@@ -0,0 +1,173 @@
+		  /* GCompris - shapes.qml
```

Comment:
fix the whole indentation of the file, there shouldn't be tabs at beginning of the lines